kick.go

```diff
@@ -0,0 +1,24 @@
+package pitaya
+
+import (
+	"context"
+
+	"github.com/topfreegames/pitaya/logger"
+	"github.com/topfreegames/pitaya/protos"
+	"github.com/topfreegames/pitaya/session"
+)
+
+// SendKickToUser sends kick to an user
+func SendKickToUser(uid, frontendType string) error {
+	if s := session.GetSessionByUID(uid); s != nil {
```

Comment:
You should check if the current server is a frontend server here, this call doesn't make sense for backend servers